src/components/stepforms/ToDateStepForm.vue

```diff
@@ -6,15 +6,31 @@
       v-model="editedStep.column"
       name="Column to convert:"
       :options="columnNames"
-      placeholder="Add columns"
-      data-path=".columns"
+      placeholder="Add column"
+      data-path=".column"
       :errors="errors"
     />
-    <InputTextWidget
-      v-if="this.translator === 'mongo40'"
-      class="dateFormat"
-      v-model.number="editedStep.format"
+    <AutocompleteWidget
+      v-if="translator === 'mongo40'"
       name="Date format:"
+      class="format"
+      :value="format"
+      @input="updateStepFormat"
+      :options="formats"
+      placeholder="Date format"
+      :trackBy="`format`"
+      :label="`label`"
+      :withExample="true"
+    />
+    <InputTextWidget
+      v-if="
+        translator === 'mongo40' &&
+          editedStep.format !== undefined &&
+          !datePresets.includes(editedStep.format)
+      "
+      class="customFormat"
+      v-model="editedStep.format"
+      name="Custom date format:"
       placeholder="%Y-%m-%d"
```

Comment:
You can use the `translator` here for example (really not mandatory) in the placeholder to say: `enter a {translator} date specification`

Reply:
Oh yes it's nice!